### PR TITLE
Update links in Concepts docs

### LINKED ISSUE #166: Update links that point to deprecated repositories
## Done Criteria
Some repositories, files, or lines of code referenced as links in docs pages may point to out of date repos. We should update these links to point to the correct location

## Tasks
- [x] Update links in Stream Multiplexing
- [x] Identify other out of date links and update

### PR DESCRIPTION
**Pull request by DannyS03**
closes #166

### CODE REVIEW

File: content/concepts/stream-multiplexing.md [hunk 1]
```diff
@@ -58,23 +58,15 @@ yamux is currently supported in go and rust:
...
-
-[SPDY][wiki-spdy] is a Google-developed protocol that was the precursor to HTTP/2. SPDY implements a stream multiplexer, which is supported by some libp2p implementations:
-
-- Go: [go-smux-spdystream](https://github.com/whyrusleeping/go-smux-spdystream)
-- Javascript: [js-libp2p-spdy](https://github.com/libp2p/js-libp2p-spdy)
+quic is currently supported in go via [go-libp2p-quic-transport](https://github.com/libp2p/go-libp2p/tree/master/p2p/transport/quic).
 
 #### muxado
 
 [muxado](https://github.com/inconshreveable/muxado) is a go stream muxing library, supported by go-libp2p via [go-smux-muxado](https://github.com/whyrusleeping/go-smux-muxado).
```
Comment (mxinden) [inline content/concepts/stream-multiplexing.md:65]: Side note, is this still relevant @marten-seemann, @MarcoPolo and @julian88110?

**Review by mxinden — APPROVED** [review]
Looks good to me.

File: content/concepts/stream-multiplexing.md (hunk 1 above)
Comment (DannyS03) [reply content/concepts/stream-multiplexing.md]: @mxinden removed here and on the implementations

Comment (mxinden) [reply content/concepts/stream-multiplexing.md]: Thanks @DannyS03.

**Pull request merged**